include/aca_ovs_l3_programmer.h

```diff
@@ -20,14 +20,26 @@
 #include <string>
 
 using namespace std;
+using namespace alcor::schema;
 
+// port id is stored as the key to ports table
 struct port_table_entry {
   string virtual_ip;
   string virtual_mac;
-  string neighbor_host_dvr_mac;
+  string host_ip;
 };
 
-struct subnet_table_entry {
+// routing rule id is stored as the key to routing_rules table
+struct routing_rule_table_entry {
+  string destination; // destination IP, could be 154.12.42.24/32 (host address) or 0.0.0.0/0 (network address)
+  string next_hop_ip;
+  uint priority;
+  alcor::schema::DestinationType destination_type;
+  string next_hop_mac;
```

Comment:
[Minor] Can we reorder it like the following:
destination, destination_type, next_hope_ip, next_hop_mac, priority?

Reply:
fixed.